kratos/utilities/condition_number_utility.h

```diff
@@ -127,7 +146,7 @@ class ConditionNumberUtility
         pEigenSolverMin->Solve(InputMatrix, identity_matrix, eigen_values, eigen_vectors);
         const double min_lambda = eigen_values[0];
         
-        condition_number = std::abs(max_lambda)/std::abs(min_lambda); 
+        const double condition_number = std::abs(max_lambda)/std::abs(min_lambda); 
```

Comment:
what if min_lambda is zero?

Reply:
Good question, I will throw an error in that case